test/foo/lib/config/config.yaml

```diff
@@ -0,0 +1,3 @@
+VAR0: fizz
+VAR2: foo
```

Comment:
It would be more readable if keys would have names in a simple order.
How about giving them names like:
KEY1: VALUE1
KEY2: VALUE2
and so on?